src/data/claims.ts

```diff
@@ -265,12 +265,12 @@ export async function updateClaim(claimId: string, newClaimData) {
   return updatedClaim
 }
 
-export async function claimsFileAttach(claimId: string, fileId: string) {
+export async function claimsFileAttach(claimId: string, fileId: string, purpose: ClaimFilePurpose) {
   start(fileId)
 
   const data: ClaimsFileAttachRequestBody = {
     file_id: fileId,
-    purpose: undefined // TODO: get this value
+    purpose: undefined
```

Comment:
Should this have been 
```suggestion
    purpose,
```
?

Reply:
why yes, thought I had done that.